lib/components/opaque_image_view.js

```diff
@@ -0,0 +1,17 @@
+/* @flow */
+'use strict';
+
+import React from 'react-native';
+
+export default class OpaqueImageView extends React.Component {
+  render() {
```

Comment:
Yeah, the one that ships with RN doesn’t have any aspect ratio layout capabilities. Plus it doesn’t share our SDWebImage cache.